presto-docs/src/main/sphinx/release/release-0.183.rst

```diff
@@ -0,0 +1,60 @@
+=============
+Release 0.183
+=============
+
+General Changes
+---------------
+
+* Fix planning failure for queries that use ``GROUPING`` and contain aggregation expressions
+  that require implicit coercions.
+* Fix planning failure for ``LEFT JOIN`` converted to ``INNER JOIN`` without equi-conditions.
```

Comment:
Is there a better way to state "without equi-conditions"? I understand an equi-join or non-equi join, but this is confusing.